tests/load/dependencies.go

```diff
@@ -27,7 +27,7 @@ type Listener[T comparable] interface {
 
 // Tracker keeps track of the status of transactions.
 // This must be thread-safe, so it can be called in parallel by the issuer(s) or orchestrator.
-type Tracker[T comparable] interface {
+type Tracker[T any] interface {
```

Comment:
I think this needs to be comparable so implementation can track T in maps etc. - as in it's more for the implementation aspect of it, true, but I think it's good to restrict this.